static/app/utils/discover/charts.tsx

```diff
@@ -49,7 +55,11 @@ export function axisLabelFormatter(
     case 'percentage':
       return formatPercentage(value, 0);
     case 'duration':
-      return axisDuration(value);
+      let durationUnit;
+      if (range) {
+        durationUnit = categorizeDuration((range.max - range.min) * 0.5);
```

Comment:
curious why we're multiplying the diff by 0.5 here?

Reply:
Whoops, It should be (range.max + range.min) * 0.5, this would give the number exactly between the two. I just realized, i'll need to dynamically set the range.max based on what percentiles are selected too
<img width="387" alt="image" src="https://user-images.githubusercontent.com/44422760/177385273-187beee9-ead4-4cf7-9be5-fcb694ae1c0e.png">

right now range.max = max(p90) value, need to consider the case when p90 is not selected